ha/core/event_manager/event_manager.py

```diff
@@ -104,23 +275,27 @@ def get_events(self, component : str) -> list:
         Returns:
             list: List of events.
         """
-        pass
 
     def publish(self, component:str, event: RecoveryActionEvent) -> None:
         """
         Publish event.
-
         Args:
             component (str): Component name.
             event (RecoveryActionEvent): Action event.
         """
-        pass
 
     def message_type(self, component: str) -> str:
         """
         It returns message type name (queue name) mapped with component.
-
         Args:
             component (str): component name.
         """
-        pass
+
+ev = EventManager.get_instance()
+ev.subscribe('hare', SubscribeEvent('node:os:memory_usage', ['failed']))
+ev.subscribe('hare', SubscribeEvent('node:interface:nw', ['online']))
```

Comment:
ev.subscribe('hare', SubscribeEvent('node:interface:nw', ['online']))
I think it should be as below
ev.subscribe('hare', [SubscribeEvent('node:interface:nw', ['online']]))

second parameter is list